superset/viz.py

```diff
@@ -790,7 +790,7 @@ def get_data(self, df):
         records = df.to_dict('records')
         for metric in self.metric_labels:
             data[metric] = {
-                str(obj[DTTM_ALIAS] / 10**9): obj.get(metric)
+                str(obj[DTTM_ALIAS].value / 10**9): obj.get(metric)
```

Comment:
I feel like we've been going back and forth with the `.value` here. Maybe it flickers based on the pandas version? Are you on `0.23.1` ? It was last changed on this commit: 7b3095d6

Reply:
Yes I'm on the latest pandas version 0.23.1 and Pytho: 3.6.6.final.0 , and without `.value` you get the following error
```TypeError: unsupported operand type(s) for /: 'Timestamp' and 'int'```

Reply:
My point is that it will fail in other circumstances, I'm not sure exactly which ones. How about we try another approach? Maybe either `float(obj[DTTM_ALIAS]) / 10**9`, or perhaps checking with `hasattr(obj[DTTM_ALIAS], 'value')`?